pkgs/racket-doc/scribblings/guide/begin.scrbl

```diff
@@ -64,6 +56,22 @@ positions, instead of forming an expression, the content of
 This splicing behavior is mainly useful for macros, as we discuss
 later in @secref["macros"].
 
+The whole @racket[_body] sequence is sometimes said to have an @deftech{implicit begin}.
```

Comment:
Why sometimes? Probably best to always or never (at least in the documentation we control).

Reply:
Also, I'm not sure it's clear that the metavariable `body` here refers to internal-definition contexts, so maybe `The whole @racket[_body] sequence of an internal-definition context`?

Reply:
One complication is that the Racket Guide doesn't really define what internal-definition context is, so I've been avoiding it. But perhaps we should define it, perhaps in the section "Internal Definitions".